app/templates/settings/general.hbs

```diff
@@ -71,7 +71,7 @@
             </div>
             <div class="gh-setting-action gh-setting-action-smallimg">
                 {{#if model.icon}}
-                    <img class="blog-icon" src="{{model.icon}}" alt="icon" data-test-icon-img>
+                    <img class="blog-icon" src="{{model.icon}}" onClick={{action "triggerFileDialog"}} alt="icon" data-test-icon-img>
```

Comment:
Standard native event names (even when preceded with `on` to create event handlers) should always be lowercase, e.g. `onclick`. See https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Event_handlers and https://developer.mozilla.org/en-US/docs/Web/Events. camelCase event handlers will work in some browsers but it's recommended to follow the standards.

The only time event names are camelCased in Ember are when using Ember's built-in or custom added event handler methods inside of a component, e.g.:

```
export default Ember.Component.extend({
  click() {
    return false;
  },

  doubleClick() {
     alert("DoubleClickableComponent was double clicked!");
  }
});
```

A full list of Ember event handlers is in the docs https://guides.emberjs.com/v2.13.0/components/handling-events/#toc_event-names